frontend/ui/elements/font_settings.lua

```diff
@@ -6,7 +6,8 @@ local _ = require("gettext")
 --[[ Font settings for desktop linux and mac ]]--
 
 local function getUserDir()
-    return os.getenv("HOME").."/.local/share/fonts"
+    local home = os.getenv("HOME") or ""
+    return home.."/.local/share/fonts"
```

Comment:
@pazos This workaround isn't quite right obviously, except for preventing a crash.

Also, http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html#tag_08_03

> The system shall initialize this variable at the time of login to be a pathname of the user's home directory. See <pwd.h>.

I'm not entirely sure what's happening here since it should just exist. But it's not related to this PR other than that this code path wasn't hit during the unit tests until now.